core/modules/system/system.module

```diff
@@ -3496,6 +3496,7 @@ function system_autoload_info() {
     'Archive_Tar' => 'system.tar.inc',
     'ModuleUpdater' => 'system.updater.inc',
     'ThemeUpdater' => 'system.updater.inc',
+    'BackdropSetMessageTest' => 'tests/system.test',
```

Comment:
Hi @serundeputy! You're right, test classes don't need to be added to `hook_autoload_info()`. The classes are loaded individually based on the class name and file path from the `*.tests.info` file.